src/hooks/useProject.js

```diff
@@ -66,4 +81,36 @@ export const useProject = ({
     const data = defaultPythonProject;
     dispatch(setProject(data));
   }, [projectIdentifier, cachedProject, i18n.language, accessToken]);
+
+  useEffect(() => {
+    if (code && loading === "success") {
+      const defaultName = project.project_type === "html" ? "index" : "main";
+      const defaultExtension = project.project_type === "html" ? "html" : "py";
```

Comment:
I like this dynamic switching, but I think I'm not understanding something - if we only load assets, there will presumably be no `project` and thus no `project_type` which should therefore default to python.

Is there a way to set a default project type e.g. via props?

Reply:
Yes, I'm sure we could do that, although I left it out for now as I'm not sure it's needed for this. I put the switching in so it can override a project set by the identifier as well (so where main/index already exists)

Reply:
Roger!